src/components/homepage/Homepage.tsx

```diff
@@ -51,7 +49,7 @@ const Homepage: React.FC<{}> = () => {
         >
           <Link
             className={classes.itemLink}
-            to="/editprofile"
+            to="/edit/profile"
```

Comment:
don't forget to update this route,
also, it's a good practice to define constants for route names, then you only have to change one variable to update the route name